agent/src/capability/configstore.cpp

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2016 Pelagicore AB
+ *
+ * Permission to use, copy, modify, and/or distribute this software for
+ * any purpose with or without fee is hereby granted, provided that the
+ * above copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+ * WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+ * BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+ * OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+ * WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+ * ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+ * SOFTWARE.
+ *
+ * For further information see LICENSE
+ */
+
+#include "configstore.h"
+
+
+ConfigStore::ConfigStore()
+{
+}
+
+ConfigStore::~ConfigStore()
+{
+}
+
+    static std::vector<GatewayConfiguration> capFromFile(std::string capID, std::string &configFilePath);
+    static std::vector<GatewayConfiguration> capFromDir(std::string capID, std::string &configDirPath);
```

Comment:
Copy paste error from the h file. I saw this myself. I also saw that my cpp and h files are not aligned at all. Sorry about that.